src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java

```diff
@@ -1073,26 +1073,29 @@ public static ItemColor leavesItemColor() {
             .initialProperties(() -> Blocks.STRIPPED_SPRUCE_LOG)
             .lang("Stripped Rubber Log")
             .blockstate((ctx, provider) -> provider.logBlock(ctx.get()))
-            .tag(ItemTags.LOGS_THAT_BURN, BlockTags.MINEABLE_WITH_AXE)
+            .tag(BlockTags.MINEABLE_WITH_AXE)
             .simpleItem()
+            .tag(ItemTags.LOGS_THAT_BURN)
```

Comment:
also nope. could you please, just please, actually pull the code and look at what's wrong?

Reply:
should work now. hopefully. crossing all digits I can.